backend/capellacollab/projects/crud.py

```diff
@@ -37,3 +41,13 @@ def create_project(db: Session, name: str) -> DatabaseProject:
 def delete_project(db: Session, name: str) -> None:
     db.query(DatabaseProject).filter(DatabaseProject.name == name).delete()
     db.commit()
+
+
+def get_slug(db: Session, slug: str) -> DatabaseProject:
+    project = db.query(DatabaseProject).filter(DatabaseProject.slug == slug).first()
+    assert project is not None
+    return project
+
+
+def get_id(db: Session, id: int):
```

Comment:
The return type is missing: `DatabaseProject`

Reply:
Actually all the routes using the ID to fetch a project are unused, that’s also why they are not well finished. I’m removing them.